src/libs/ReportActionsUtils.ts

```diff
@@ -616,6 +616,26 @@ function isNotifiableReportAction(reportAction: OnyxEntry<ReportAction>): boolea
     return actions.includes(reportAction.actionName);
 }
 
+/**
+ * Helper method to determine if the provided accountID has made a request on the specified report.
+ *
+ * @param reportID
+ * @param currentAccountID
+ * @returns
+ */
+function hasRequestFromPayer(reportID: string, currentAccountID: number): boolean {
```

Comment:
This method doesn't actually check if the currentAccount is the payer, so I think this name is more accurate. 
```suggestion
function hasRequestFromCurrentAccount(reportID: string, currentAccountID: number): boolean {
```